jdbc-driver/src/test/java/com/google/cloud/dataproc/DataprocDriverTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.dataproc;
```

Comment:
Gradle/Maven group ID doesn't map directly to the Java package, even though we switched to `com.google.cloud.dataproc` group, we still should use `com.google.cloud.dataproc.jdbc` Java package for this project.